packages/flame/lib/src/game/game_widget/game_widget.dart

```diff
@@ -22,11 +23,16 @@ typedef OverlayWidgetBuilder<T extends Game> = Widget Function(
   T game,
 );
 
+typedef GameCreate<T extends Game> = T Function();
```

Comment:
```suggestion
typedef GameBuilder<T extends Game> = T Function();
```
I think it should be a noun here instead of verb.

Reply:
Not a builder. On the framework (consequently on most packages as well), the suffix builder usually means methods that run in the build phase and generate widget instances. That is not the case here. `GameFactory` or `GameCreator` would also work?

Reply:
I don't think that is true, the builder pattern is a much older design pattern than Flutter, and it is used as such in many places in the Flutter world too. But `GameFactory` works too, maybe even better!
https://medium.com/flutter-community/flutter-design-patterns-18-builder-cdc90b222724